examples/options/src/track_finding.cpp

```diff
@@ -66,9 +66,11 @@ track_finding::track_finding() : interface("Track Finding Options") {
         po::value(&m_config.max_num_skipping_per_cand)
             ->default_value(m_config.max_num_skipping_per_cand),
         "Maximum allowed number of skipped steps per candidate");
-    m_desc.add_options()("particle-hypothesis",
-                         po::value(&m_pdg_number)->default_value(m_pdg_number),
-                         "PDG number for the particle hypothesis");
```

Comment:
Is there a a reason to remove this argument?

Reply:
Mistake - will fix it